### PR TITLE
Bug 1197735 - [Bluetooth] proper process _onDefaultAdapterChanged stat, r=tim

### PR DESCRIPTION
**Pull request by gasolin**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f7b54a8f9992aadc8092751c48ad4d96adfde74e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=26b6df4db62e6a3effa6e10230fe4fcc64d8a715

File: apps/bluetooth/js/modules/transfer_manager.js [hunk 1]
```diff
@@ -174,6 +175,9 @@ define(function(require) {
     _onBluetoothEnabledChanged:
     function bttm__onBluetoothEnabledChanged(enabled) {
       Debug('_onBluetoothEnabledChanged(): enabled = ' + enabled);
+      if (enabled === undefined) {
```
Comment (timdream) [inline apps/bluetooth/js/modules/transfer_manager.js]: Let's use `switch(enabled) { ... }` to handle the triple states.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a57b058a7c4e1fe124f50d6623f6637a568fd2d1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8845b304ae0d492c43f8649a7c1d2f3229da5b12

**Pull request merged**